php/libraries/Data/DAO.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace LORIS\Data;
+
+interface DAO
```

Comment:
I'm not crazy about all the acronyms. How about. They're effectively data model objects, so how about

```
namespace LORIS\Data\Models;

interface AccessObject {
[...]
}
```